packages/axelar-soroban-std/src/testutils.rs

```diff
@@ -73,41 +73,32 @@ where
 
 #[macro_export]
```

Comment:
Add doc on how to use this. You can add an executable Rust example in the doc, see the derive macro crate for an example